scripts/createSolutions.js

```diff
@@ -1,6 +1,7 @@
 const cheerio = require('cheerio');
 const fs = require('fs');
 const path = require('path');
+const sharp = require("sharp");
```

Comment:
The `sharp` dependency is missing from your `package.json` file. Please check-in your `package.json` file also after adding the `sharp` dependency.

Reply:
Package.json was added and the dependency was adjusted.